src/lib/utils/balancer/api.ts

```diff
@@ -0,0 +1,10 @@
+import { configService } from '@/services/config/config.service';
+
+export function isBalancerApiDefined() {
+  const defined = configService.network.balancerApi;
+  if (!defined)
+    console.log(
+      `Skipping balancer api provider in your current network (${configService.network.chainName})`
+    );
+  return defined;
+}
```

Comment:
I don't think we need to log here and this should return a boolean.
```suggestion
export const isBalancerApiDefined = !!configService.network.balancerApi
```